.eslintrc.js

```diff
@@ -101,5 +102,6 @@ module.exports = {
         'no-use-before-define': 'off',
         '@typescript-eslint/no-use-before-define': ['error'],
         'require-await': ['error'],
+        '@typescript-eslint/triple-slash-reference': 'off',
```

Comment:
Maybe better to add an exception for just the global.d.ts file instead of disabling the whole rule.

Reply:
cf1975a